openstack_exporter/collectors/cinderbackend.py

```diff
@@ -250,136 +273,97 @@ def collect(self):
+                shard_name, backend, pool_name
+            )
 
-            if can_overcommit:
-                g = GaugeMetricFamily('cinder_max_oversubscription_ratio',
-                                      'Cinder max overcommit ratio',
-                                      labels=['backend', 'pool', 'shard'])
-                g.add_metric([backend, pool_name, shard_name],
-                             value=data['max_over_subscription_ratio'])
-                yield g
-
-                g = GaugeMetricFamily('cinder_overcommit_ratio',
-                                      'Cinder Overcommit ratio',
-                                      labels=['backend', 'pool', 'shard'])
-                g.add_metric([backend, pool_name, shard_name],
-                             value=data['overcommit_ratio'])
-                yield g
-
-            g = GaugeMetricFamily('cinder_reserved_precentage',
-                                  'Cinder Reserved Space Percentage',
-                                  labels=['backend', 'pool', 'shard'])
-            g.add_metric([backend, pool_name, shard_name],
-                         value=data['reserved_percentage'])
-
-            yield g
-
-            LOG.debug('({}/{}/{})-per_volume_gigabytes = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                quota_obj.per_volume_gigabytes
-            ))
-
-            LOG.debug('({}/{}/{})-provisioning_type = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                provisioning_type
-            ))
-            LOG.debug('({}/{}/{})-total_capacity_gb = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                data['total_capacity_gb']
-            ))
-            LOG.debug('({}/{}/{})-available_capacity_gb = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                data['available_capacity_gb']
-            ))
-            LOG.debug('({}/{}/{})-free_capacity_gb = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                data['free_capacity_gb']
-            ))
-            LOG.debug('({}/{}/{})-virtual_free_capacity_gb = {}'.format(
-                shard_name, data['backend'], data['pool'],
-                data['virtual_free_capacity_gb']
-            ))
-            LOG.debug('({}/{}/{})-allocated_capacity_gb = {}'.format(
-                shard_name, caps['volume_backend_name'], data['pool'],
-                data['allocated_capacity_gb']
-            ))
+            yield self.add_gauge_metric_gauge(
+                'cinder_total_capacity_gib',
+                'Cinder total capacity in Gib',
+                data.get('total_capacity_gb'),
+                shard_name, backend, pool_name
+            )
+
+            yield self.add_gauge_metric_gauge(
+                'cinder_available_capacity_gib',
+                'Cinder available capacity in GiB',
+                data['available_capacity_gb'],
```

Comment:
Why do we do a `data.get('total_capacity_gb')` but assume `available_capacity_gb` is always there? This was already inconsistent before this commit, but at least `available_capacity_gb` was also a `get()` before.